templates/pulse_schedule.html

```diff
@@ -54,19 +52,21 @@
                 } else {
                     upmatch.innerHTML =
                         "Team " +
-                        myTeam +
+                        data.team_key +
                         " does not have any upcomming matches";
                 }
                 html += "<br>";
             }
         }
         upmatch.innerHTML = html;
-        nextMatch = upmatches[0];
+        nextMatch = (upmatches.length > 0)
+            ? upmatches[0]
+            : "No upcomming matches";
         let nextColor = "ERROR";
         for (let i = 0; i < nextMatch["blueTeams"].length; i++) {
-            if (nextMatch["blueTeams"][i] == myTeam) {
+            if (nextMatch["blueTeams"][i] == data.team_key) {
                 nextColor = "blue";
-            } else if (nextMatch["redTeams"][i] == myTeam) {
+            } else if (nextMatch["redTeams"][i] == data.team_key) {
```

Comment:
```suggestion
            } else if (nextMatch["redTeams"][i] === data.team_key) {
```

Reply:
shouldn't it be == for if it needs to compare number and string?

Reply:
Am I blind? Yes. Thank for catching my stupidity.